packages/block-library/src/block/edit.js

```diff
@@ -151,20 +151,20 @@ function getContentValuesFromInnerBlocks( blocks, defaultValues ) {
 			content,
 			getContentValuesFromInnerBlocks( block.innerBlocks, defaultValues )
 		);
-		const blockId = block.attributes.metadata?.id;
-		if ( ! hasOverridableAttributes( block ) || ! blockId ) continue;
+		const metadataName = block.attributes.metadata?.name;
+		if ( ! metadataName || ! hasOverridableAttributes( block ) ) continue;
```

Comment:
Nit: Same nit as above relating to the [WordPress JavaScript Coding Standards](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/javascript/) state:

> if/else/for/while/try blocks should always use braces, and always go on multiple lines